### PR TITLE
Added class SearchFilter (DTO). Added relevant attributes, getter and…

### PR DESCRIPTION
**Pull request by Hackerlacka**
… setter methods.

### CODE REVIEW

File: src/main/java/se/lth/base/server/data/SearchFilter.java [hunk 1]
```diff
@@ -0,0 +1,50 @@
+package se.lth.base.server.data;
+
+import java.sql.Timestamp;
+
+public class SearchFilter {
+
+    private final int searchFilterId, userId;
```
Comment (jost95) [inline src/main/java/se/lth/base/server/data/SearchFilter.java]: Separate attribute decorations

File: src/main/java/se/lth/base/server/data/SearchFilter.java [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+    public int getUserId() {
+        return userId;
+    }
+
+    public String getStart() {
+        return start;
+    }
+
+    public void setStart(String start) {
```
Comment (jost95) [inline src/main/java/se/lth/base/server/data/SearchFilter.java]: Not needed

File: src/main/java/se/lth/base/server/data/SearchFilter.java [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+        this.start = start;
+    }
+
+    public String getStop() {
+        return stop;
+    }
+
+    public void setStop(String stop) {
```
Comment (jost95) [inline src/main/java/se/lth/base/server/data/SearchFilter.java]: Not needed

File: src/main/java/se/lth/base/server/data/SearchFilter.java [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+        this.stop = stop;
+    }
+
+    public Timestamp getDepartureTime() {
+        return departureTime;
+    }
+
+    public void setDepartureTime(Timestamp departureTime) {
```
Comment (jost95) [inline src/main/java/se/lth/base/server/data/SearchFilter.java]: Not needed

**Comment by Hackerlacka** [comment]
I was looking at the class DriveUser when I decided to put some of the attributes on the same row. Maybe that class should be updated as well?

**Pull request merged**